DataFormats/Detectors/ZDC/include/DataFormatsZDC/BCData.h

```diff
@@ -0,0 +1,51 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef _ZDC_BC_DATA_H_
+#define _ZDC_BC_DATA_H_
+
+#include "CommonDataFormat/InteractionRecord.h"
+#include "CommonDataFormat/RangeReference.h"
+#include "ZDCBase/Constants.h"
+#include <Rtypes.h>
+
+/// \file DCData
```

Comment:
Probably this should be BC and not DC? Is this class ZDC specific? Maybe a slightly more specific name than BCData might be good. Something like "TriggerData"?

Reply:
No, this is actually just a stored BC + reference on the stored channels, can be triggered on simply stored for the baseline or pile-up checks